test/e2e-website/material-current.spec.ts

```diff
@@ -10,15 +10,12 @@ test.describe.parallel('Material docs', () => {
 
     const anchors = page.locator('[aria-label="Page table of contents"] ul a');
 
-    const anchorTexts = await anchors.allTextContents();
-
-    await Promise.all(
-      anchorTexts.map((text, index) => {
-        return expect(anchors.nth(index)).toHaveAttribute(
-          'href',
-          `/getting-started/installation/#${kebabCase(text)}`,
-        );
-      }),
+    const firstAnchor = await anchors.first();
+    const textContent = await firstAnchor.textContent();
+
+    await expect(firstAnchor).toHaveAttribute(
+      'href',
+      `/getting-started/installation/#${kebabCase(textContent || '')}`,
```

Comment:
no need to test all the links, the first is enough to make sure that TOC has `/getting-started/installation/*`